2021-05-19 22:13:36,855 INFO    MainThread:13124 [wandb_setup.py:_flush():69] setting env: {}
2021-05-19 22:13:36,855 INFO    MainThread:13124 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-19 22:13:36,855 INFO    MainThread:13124 [wandb_init.py:_log_setup():336] Logging user logs to /home/kukjin/Study/RL/소프트웨어 융합 캡스톤 디자인/ApexDQFD/wandb/run-20210519_221336-2u6rnky6/logs/debug.log
2021-05-19 22:13:36,855 INFO    MainThread:13124 [wandb_init.py:_log_setup():337] Logging internal logs to /home/kukjin/Study/RL/소프트웨어 융합 캡스톤 디자인/ApexDQFD/wandb/run-20210519_221336-2u6rnky6/logs/debug-internal.log
2021-05-19 22:13:36,856 INFO    MainThread:13124 [wandb_init.py:_jupyter_setup():287] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fb8d86ada50>
2021-05-19 22:13:36,856 INFO    MainThread:13124 [wandb_init.py:init():369] calling init triggers
2021-05-19 22:13:36,856 INFO    MainThread:13124 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-19 22:13:36,856 INFO    MainThread:13124 [wandb_init.py:init():418] starting backend
2021-05-19 22:13:36,856 INFO    MainThread:13124 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-19 22:13:36,859 INFO    MainThread:13124 [backend.py:ensure_launched():123] starting backend process...
2021-05-19 22:13:36,873 INFO    MainThread:13124 [backend.py:ensure_launched():128] started backend process with pid: 13892
2021-05-19 22:13:36,873 INFO    MainThread:13124 [wandb_init.py:init():423] backend started and connected
2021-05-19 22:13:36,887 INFO    MainThread:13124 [wandb_init.py:init():465] updated telemetry
2021-05-19 22:13:36,888 INFO    MainThread:13124 [wandb_init.py:init():484] communicating current version
2021-05-19 22:13:41,888 INFO    MainThread:13124 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-19 22:13:46,529 WARNING MainThread:13124 [wandb_init.py:init():764] interrupted
Traceback (most recent call last):
  File "/home/kukjin/anaconda3/envs/minerl/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 744, in init
    run = wi.init()
  File "/home/kukjin/anaconda3/envs/minerl/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 498, in init
    ret = backend.interface.communicate_run(run, timeout=30)
  File "/home/kukjin/anaconda3/envs/minerl/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 652, in communicate_run
    return self._communicate_run(run, timeout=timeout)
  File "/home/kukjin/anaconda3/envs/minerl/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 640, in _communicate_run
    resp = self._communicate(req, timeout=timeout)
  File "/home/kukjin/anaconda3/envs/minerl/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 533, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/kukjin/anaconda3/envs/minerl/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 82, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/kukjin/anaconda3/envs/minerl/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/home/kukjin/anaconda3/envs/minerl/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2021-05-19 22:13:59,680 INFO    MainThread:13124 [wandb_settings.py:_apply_source_login():724] setting login settings: {}
2021-05-19 22:13:59,685 INFO    MainThread:13124 [wandb_setup.py:_flush():69] setting env: {}
2021-05-19 22:13:59,685 INFO    MainThread:13124 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-19 22:13:59,685 INFO    MainThread:13124 [wandb_init.py:_log_setup():336] Logging user logs to /home/kukjin/Study/RL/소프트웨어 융합 캡스톤 디자인/ApexDQFD/wandb/run-20210519_221359-2gumah7l/logs/debug.log
2021-05-19 22:13:59,685 INFO    MainThread:13124 [wandb_init.py:_log_setup():337] Logging internal logs to /home/kukjin/Study/RL/소프트웨어 융합 캡스톤 디자인/ApexDQFD/wandb/run-20210519_221359-2gumah7l/logs/debug-internal.log
2021-05-19 22:13:59,686 INFO    MainThread:13124 [wandb_init.py:init():369] calling init triggers
2021-05-19 22:13:59,686 INFO    MainThread:13124 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-19 22:13:59,686 INFO    MainThread:13124 [wandb_init.py:init():418] starting backend
2021-05-19 22:13:59,686 INFO    MainThread:13124 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-19 22:13:59,688 INFO    MainThread:13124 [backend.py:ensure_launched():123] starting backend process...
2021-05-19 22:13:59,732 INFO    MainThread:13124 [backend.py:ensure_launched():128] started backend process with pid: 13972
2021-05-19 22:13:59,733 INFO    MainThread:13124 [wandb_init.py:init():423] backend started and connected
2021-05-19 22:13:59,746 INFO    MainThread:13124 [wandb_init.py:init():465] updated telemetry
2021-05-19 22:13:59,747 INFO    MainThread:13124 [wandb_init.py:init():484] communicating current version
2021-05-19 22:14:04,747 INFO    MainThread:13124 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-19 22:14:34,748 INFO    MainThread:13124 [interface.py:_communicate_run():642] couldn't get run from backend
2021-05-19 22:14:34,748 ERROR   MainThread:13124 [wandb_init.py:init():501] backend process timed out
2021-05-19 22:14:34,748 ERROR   MainThread:13124 [wandb_init.py:init():510] encountered error: Error communicating with wandb process
try: wandb.init(settings=wandb.Settings(start_method='fork'))
or:  wandb.init(settings=wandb.Settings(start_method='thread'))
For more info see: https://docs.wandb.ai/library/init#init-start-error
2021-05-19 22:14:39,768 INFO    MainThread:13124 [wandb_init.py:teardown():187] tearing down wandb.init
